docs/upgrade/upgrade-security.asciidoc

```diff
@@ -27,6 +27,12 @@ Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-
 
 NOTE: You do not need to shut down your {agent}s or endpoints to upgrade the {stack}.
 
+[float]
+[[update-prebuilt-rules]]
+=== Update prebuilt detection rules before upgrading to {stack} 8.0 or 8.1
+
+If you're upgrading to {stack} 8.0 or 8.1, you must update your Elastic prebuilt detection rules while your {stack} is at 7.17, before upgrading to 8.x. To update Elastic prebuilt rules, go to the Rules page and select *Update _x_ Elastic prebuilt rules*. This ensures that you have the latest prebuilt rules before upgrading the {stack} to 8.x.
```

Comment:
would it be helpful to specify 8.0.* and 8.1.*?

Reply:
It's probably understood that `8.0` = `8.0.x` (that seems to be the general convention I've seen in most docs, I think?), but it also doesn't hurt to include the `x` to make it super clear. This also incorporates your other edit below:
```suggestion
If you're upgrading to {stack} 8.0.x or 8.1.x, you must update your Elastic prebuilt detection rules while your {stack} is at 7.17, before upgrading the {stack}. To update Elastic prebuilt rules, go to the Rules page and select *Update _x_ Elastic prebuilt rules*. This ensures that you have the latest prebuilt rules before upgrading the {stack} to 8.x.
```